x/reward/keeper/genesis.go

```diff
@@ -0,0 +1,28 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/xpladev/xpla/x/reward/types"
+)
+
+func (k Keeper) InitGenesis(ctx sdk.Context, data types.GenesisState) {
+	k.SetParams(ctx, data.Params)
+
+	// check if the module account exists
```

Comment:
check this

Reply:
https://github.com/xpladev/xpla/pull/18/commits/3abdc41eaae736f6f9220a074bbad795be70d48c